web/libs/editor/src/tags/object/Video/Video.js

```diff
@@ -198,28 +201,46 @@ const Model = types
       },
 
       addRegion(data) {
-        const control = self.videoControl() ?? self.control();
+        const control = self.videoControl ?? self.timelineControl;
```

Comment:
If we have more than one control in the config we do not get the correct one here.

Reply:
Changed control getters to be explicit and split their usage — now it's `addVideoRegion()` and `addTimelineRegion()`